### PR TITLE
[tests-only][full-ci]Add then step to suite apiWebdavOperations 

### PR DESCRIPTION
**Pull request by SagarGi**
## Description
This PR adds `Then` steps to existing API test scenarios to better check the results of `When` steps.

**Suites covered:**
- `apiWebdavOperations`

## Related Issue

- Part of https://github.com/owncloud/core/issues/39512

## How Has This Been Tested?

- Locally
- CI

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: tests/acceptance/features/apiWebdavOperations/listFiles.feature [hunk 1]
```diff
@@ -149,6 +200,22 @@ Feature: list files
       | /simple-folder1/simple-folder2/welcome.txt                   |
       | /simple-folder1/simple-folder2/simple-folder3                |
       | /simple-folder1/simple-folder2/simple-folder3/simple-folder4 |
+  @notToImplementOnOCIS @issue-ocis-2079
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavOperations/listFiles.feature:203]:
```suggestion
    @notToImplementOnOCIS @issue-ocis-2079
```

File: tests/acceptance/features/apiWebdavOperations/listFiles.feature [hunk 2]
```diff
@@ -149,6 +200,22 @@ Feature: list files
...
+    Examples:
+      | dav_version |
+      | old         |
+
+    Examples:
+      | dav_version |
+      | new         |
```
Comment (kiranparajuli589) [inline tests/acceptance/features/apiWebdavOperations/listFiles.feature:210]: let's add an example for personal spaces too.

Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavOperations/listFiles.feature:210]: also, look for other newly added scenarios and add personal space examples

**Review by phil-davis — APPROVED** [review]
LGTM

**Pull request merged**